packages/apidom-ns-openapi-3-1/src/refractor/plugins/normalize-operation-ids.ts

```diff
@@ -116,10 +133,27 @@ const plugin =
           },
         },
         OperationElement: {
-          enter(operationElement: OperationElement) {
+          enter(
+            operationElement: OperationElement,
+            key: string | number,
+            parent: Element | undefined,
+            elementPath: (string | number)[],
```

Comment:
Not sure what to name this, considering that we get `path` from `paths` in this one:
```js
const path = last(paths) as string;
````

Reply:
I'd rename the top level paths to pathTemplates and then use paths in visitor method arguments